kimhaejung/app.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const cors = require("cors");
+const dotenv = require("dotenv");
+const { DataSource } = require("typeorm");
+const app = express();
+const logger = require("morgan");
+
+dotenv.config();
+
+app.use(cors());
+app.use(logger("combined"));
+
+app.get("/ping", function (req, res, next) {
+  res.json({ message: "pong" });
+});
+
+const myDataSource = new DataSource({
```

Comment:
myDataSource 보다는 `appDataSource` , `mysqlDataSource` 혹은 간결하게 `database`로 변수명을 변경해주세요. “my”라는 용어는 예제 코드에서는 사용할 수 있으나 실제 코드에서는 피해주세요.

```suggestion
const appDataSource = new DataSource({
```